packages/core/src/trigger_api/keep_fresh.ts

```diff
@@ -56,6 +83,16 @@ export function keepFresh<Params>(
 
   triggers.push(...triggerEvents);
 
+  const enabledParamStores = [query.$enabled];
+  if (config.enabled !== undefined) {
+    enabledParamStores.push(config.enabled);
+  }
+
+  const $enabled = combine(
```

Comment:
🤓 I believe it's better to use a helper `every` from  `packages/core/src/libs/patronus`. It can improve code readability and reduce final bundle size.

Reply:
I'm not very familiar with patronum api yet. Thanks, that's really better

Reply:
Have to mention that it is not `patronum`, it is a local restricted version.